CHANGES.txt

```diff
@@ -17,10 +17,18 @@ Backwards incompatible changes
 - Drop the ``zope.app.testing`` extra introduced in version 2.0.0 as
   it dropped its special ``zope.testbrowser`` support.
 
+- Drop ``.browser.ExtendedTestBrowser.pretty_print`` as its requirements are
+  deprecated or even removed from Python's StdLib.
```

Comment:
(I've never seen the stdlib capitalized this way.)

Reply:
Asking Google for `python stdlib` shows me this kind of capitalisation in the majority on the first three pages.